### ISSUE TITLE
Make @extend available over multiple files

### ISSUE THREAD

**Issue by julkue** [sass/sass#1429]
Hi,

it is a problem if you want to extend a class in another file, imported with `@import`. This will not work. Please make it available over multiple files so that you can extend classes from a master-file.

**Comment by lolmaus** [comment]
It always worked for me.

**Comment by julkue** [comment]
See this answer in Stackoverflow:
http://stackoverflow.com/a/11514620/3894981

**Comment by lolmaus** [comment]
Are you not using partials?

Also, why are you using media queries via HTML instead of CSS? Are you supporting some outdated version of IE?

**Comment by cimmanon** [comment]
@julmot You did notice that's from 2012, right?  I find it hard to believe that was true even back then.  Are you sure your problem isn't this?  http://stackoverflow.com/questions/17976140/false-positive-undefined-variable-error-when-compiling-scss

**Comment by julkue** [comment]
@cimmanon @lolmaus

Unfortunately I found out, that it was my fault. I extended a element which has only a background-image in it. After searching for the .test-class which extends the other class I did not found it on the correct position. But after searching again now, I found out that it will append the selector in the position where the extended class is defined, not in the position where I defined the .test-class. So I am very sorry!
